x-pack/plugins/uptime/public/components/monitor/monitor_title.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiFlexGroup, EuiFlexItem, EuiSpacer, EuiTitle } from '@elastic/eui';
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { useMonitorId } from '../../hooks';
+import { monitorStatusSelector } from '../../state/selectors';
+import { EnableMonitorAlert } from '../overview/monitor_list/columns/enable_alert';
+import { Ping } from '../../../common/runtime_types/ping';
+import { useBreadcrumbs } from '../../hooks/use_breadcrumbs';
+
+const isAutogeneratedId = (id: string) => {
+  const autoGeneratedId = /^auto-(icmp|http|tcp|browser)-0X[A-F0-9]{16}.*/;
```

Comment:
@andrewvc You mentioned that at this point in the regex, you're content to call it an autogenerated id, so I added `.*` to account for anything else after the established regex. 

Will the letters `A-F` always be capitalized? Or do we need to add `a-f` as well?

Here is it working locally. 
<img width="1091" alt="Screen Shot 2020-12-17 at 2 59 48 PM" src="https://user-images.githubusercontent.com/11356435/102538847-da71a200-407a-11eb-84c8-84fa0d126408.png">
<img width="893" alt="Screen Shot 2020-12-17 at 2 59 55 PM" src="https://user-images.githubusercontent.com/11356435/102538850-db0a3880-407a-11eb-9e6c-4ba28eeac7e2.png">

Reply:
They will always be caps, so that's fine!

Reply:
Works like a charm now by the way.